pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,22 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	// +optional
+	Config *PrometheusConfiguration `json:"config,omitempty"`
+}
+
+// Config  is the the desired state of Prometheus Configuration
+type PrometheusConfiguration struct {
+	ConfigMapRef *ConfigMapRef `json:"configMapRef,omitempty"`
```

Comment:
Please add the comment note to explain the ConfigMapRef and the requirement of the target configmap.